client/src/app/components/Cart/Cart.js

```diff
@@ -56,14 +56,14 @@ class Cart extends Component {
           <div className={styles.empty_cart}>
             <Link to="/products">
               <img
-                src="https://res.cloudinary.com/so/image/upload/v1551445634/logos/cart.gif"
+                src="https://res.cloudinary.com/sofimat/image/upload/v1551793878/empty/cart.gif"
```

Comment:
I don't like such lazy aproach.
What are u going to do if this server will expire?

Reply:
well, it's quite reliable resource, and our database is built on it

Reply:
done